build.gradle

```diff
@@ -10,6 +11,7 @@ buildscript {
     dependencies {
         classpath 'com.android.tools.build:gradle:3.5.3'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath "io.objectbox:objectbox-gradle-plugin:$objectboxVersion"
```

Comment:
Should we move this to app/build.gradle?

Reply:
This is how they told me to do it in the ObjectBox documentation